pwiz_tools/Skyline/TestFunctional/VolcanoPlotFormattingTest.cs

```diff
@@ -414,6 +416,27 @@ private void AssertVolcanoPlotCorrect(FoldChangeVolcanoPlot plot, VolcanoPlotPoi
             });
         }
 
+        private void TestMatchExpressionListDlg(FoldChangeVolcanoPlot volcanoPlot)
+        {
+
+            var formattingDlg = ShowDialog<VolcanoPlotFormattingDlg>(volcanoPlot.ShowFormattingDialog);
+            var createExprDlg = ShowDialog<CreateMatchExpressionDlg>(() =>
+            {
+                var bindingList = formattingDlg.GetCurrentBindingList();
+                var exprInfo = MATCH_EXPR_INFOS[0][0];
+                bindingList.Add(new MatchRgbHexColor(string.Empty, exprInfo.ExpectedPointsInfo.Labeled,
+                    exprInfo.ExpectedPointsInfo.Color, exprInfo.ExpectedPointsInfo.PointSymbol,
+                    exprInfo.ExpectedPointsInfo.PointSize));
+                formattingDlg.ClickCreateExpression(bindingList.Count - 1);
+            });
+            var matchExprListDlg = ShowDialog<MatchExpressionListDlg>(createExprDlg.ClickEnterList);
+            RunUI(() =>
+            {
+                // Verify that typing into the list is parsed to a REGEX
+                matchExprListDlg.proteinsTextBox.Text = @"Cas9" + '\n' + @"Hsp70";
+                Assert.AreEqual(createExprDlg.Expression, "^Cas9$|^Hsp70$");

```

Comment:
Ideally, you would have some edge/error case testing, like when you add things that don't match, or you add empty lines, etc. I can imagine wanting the matches to be case-insensitive also. Looks like you have only case-sensitive matching?